main.js

```diff
@@ -707,14 +707,31 @@ async function getAutoUpdates() {
             let id = data['id'] ? data['id'] : data['gid'];
             let type = data['r'];
             let state = data['state'];
+            let attr = data['attr'];
             let config = data['config'];
             adapter.log.debug('Websocket message: ' + JSON.stringify(data));
 
             let thing;
             let object;
             switch (type) {
                 case 'lights':
-                    await getLightState(id);
+                    if (typeof state == 'object') {
+                        adapter.log.debug("Event has state-tag");
+                        if(Object.keys(state).length > 0) {
+                            object = await getObjectByDeviceId(id, 'Lights');
+                            for (let stateName in state) {
+                                adapter.log.debug(stateName + ": "+ state[stateName]);
+                                new SetObjectAndState(id, object.value.common.name, 'Lights', stateName, state[stateNam]);                           }
```

Comment:
Is there missing an "e" at stateNam?

Reply:
Yes it is.
Copying from a nano over ssh and writing some updates manually is no good solution.

Reply:
Pushed new version with the "e" inside.